src/Simulation/CsharpGeneration/Microsoft.Quantum.CsharpGeneration.nuspec.template

```diff
@@ -23,12 +23,5 @@
   <files>
     <file src="bin\$Configuration$\**\Microsoft.Quantum.*.pdb" target="lib" />
     <file src="bin\$Configuration$\**\Microsoft.Quantum.*.xml" target="lib" />
-    
-    <file src="..\EntryPointDriver\bin\$Configuration$\**\Microsoft.Quantum.CsharpGeneration.EntryPointDriver.dll"
```

Comment:
Out of curiosity, why are these file references no longer needed?

Reply:
This PR is removing EntryPointDriver from the CsharpGeneration package, so the DLL for it should no longer be included here (it has its own package now).